website/docs/d/pi_instance.html.markdown

```diff
@@ -63,6 +63,7 @@ In addition to all argument reference list, you can access the following attribu
   - `network_id` - (String) The network ID of the instance.
   - `network_name` - (String) The network name of the instance.
   - `type` - (String) The type of the network.
+- `placement_group_id`- (String) The ID of the placement group that the instance is in if any or empty quotes `""` when not in a placement group.
```

Comment:
No need to mention about not providing placement group.
`The placement group for the server`

Reply:
Done